src/patternfly/components/Nav/docs/code.md

```diff
@@ -24,23 +24,24 @@ The navigation system relies on several different sub-components:
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-c-nav` | `<nav>` | Initiates a primary nav element |
-| `.pf-c-nav__list` | `<ul>` | Initiates default nav list |
-| `.pf-c-nav__simple-list` | `<ul>` | Initiates simple nav list |
-| `.pf-c-nav__item` | `<li>` | Initiates default nav list item |
-| `.pf-c-nav__separator` | `<li>` | Initiates list separator |
+| `.pf-c-nav` | `<nav>` | Initiates a primary nav element. |
+| `.pf-c-nav__list` | `<ul>` | Initiates default nav list. |
+| `.pf-c-nav__simple-list` | `<ul>` | Initiates simple nav list. |
+| `.pf-c-nav__item` | `<li>` | Initiates default nav list item. |
+| `.pf-c-nav__separator` | `<li>` | Initiates list separator. |
 | `.pf-c-nav__scroll-button` | `<button>` | Intitiates a nav scroll button. **Required for horizontal navs** |
-| `.pf-c-nav__link` | `<a>` | Initiates default nav list link |
-| `.pf-c-nav__section` | `<section>` | Initiates a nav section element |
-| `.pf-c-nav__section-title` | `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, `<h6>` | Initiates a nav section title |
-| `.pf-c-nav__toggle` | `<span>` | Initiates a chevron indicating expandability of a `pf-c-nav__list-link` |
+| `.pf-c-nav__link` | `<a>` | Initiates default nav list link. |
+| `.pf-c-nav__section` | `<section>` | Initiates a nav section element. |
+| `.pf-c-nav__section-title` | `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, `<h6>` | Initiates a nav section title. |
+| `.pf-c-nav__toggle` | `<span>` | Initiates a chevron indicating expandability of a `pf-c-nav__list-link`. |
 | `.pf-c-nav__toggle-icon` | `<i>` | Initiates a nav toggle icon. |
-| `.pf-m-expandable` | `.pf-c-nav__item` | Modifies for the expandable state |
-| `.pf-m-expanded` | `.pf-c-nav__item` | Modifies for the expanded state |
-| `.pf-m-hover` | `.pf-c-nav__link` | Modifies to display the link as hovered |
-| `.pf-m-focus` | `.pf-c-nav__link` | Modifies to display the link as focussed |
-| `.pf-m-current` | `.pf-c-nav__link` | Modifies for the current state |
-| `.pf-m-active` | `.pf-c-nav__link` | Modifies to display the link as active |
-| `.pf-m-disabled` | `.pf-c-nav__link` | Modifies to display the link as disabled |
+| `.pf-m-dark` | `.pf-c-nav` | Modifies the nav for the dark variation. **Note: only for use with vertical navs, and requires `.pf-m-dark` on the page component's sidebar element** |
```

Comment:
can you add the class name to the end of this sentence just to be very clear: `pf-c-page__sidebar`